controller/src/config.py

```diff
@@ -36,6 +36,7 @@ class CtrlConfig(BaseModel):
     log_level: str = "DEBUG"
     # SQL Database
     sql_connection_str: str = default_db_path
+    application_url: str = "http://localhost:8000"
```

Comment:
it is `localhost` in a docker, not sure it will work as it won't be the same as the user's localhost. Have you tried it?

Reply:
Since we are using `docker run  --net host` the docker's network it's the same as the host - no separation, so it is working. We are using this option to communicate with the application's API (on a local port) from the controller.

Reply:
superb <3